noaa/src/CamOverlayIntegration.ts

```diff
@@ -0,0 +1,31 @@
+import { CamOverlayAPI, CamOverlayOptions } from 'camstreamerlib/CamOverlayAPI';
+
+export class CamOverlayIntegration {
+    private _camOverlayApiInstances: Map<number, CamOverlayAPI> = new Map();
+
+    constructor(private _options: CamOverlayOptions) {}
+
+    private _initializeCamOverlayApi(serviceID: number) {
+        if (!this._camOverlayApiInstances.has(serviceID)) {
+            this._camOverlayApiInstances.set(serviceID, new CamOverlayAPI({ ...this._options, serviceID }));
+        }
+    }
+
+    initializeInfoTickerCamOverlayApi(serviceID: number) {
+        this._initializeCamOverlayApi(serviceID);
+    }
+
+    initializeCustomGraphicsCamOverlayApi(serviceID: number) {
+        this._initializeCamOverlayApi(serviceID);
+    }
+
+    async updateInfoTickerText(serviceID: number, text: string) {
+        if (!this._camOverlayApiInstances.has(serviceID)) return;
```

Comment:
I'm not too fond of this one-line form without brackets but if you are used to it in a frontend team..